extensions/adobe/experience/implementations.schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$id": "https://ns.adobe.com/xdm-extensions/experience/analytics/analytics-extensions",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Experience Cloud ExperienceEvent Implementations",
+  "type": "object",
+  "meta:extensible": true,
```

Comment:
I would not make an extension itself extensible.

Reply:
Good point.